resources/js/views/gallery-panels/Favourites.vue

```diff
@@ -0,0 +1,121 @@
+<template>
+	<div class="h-svh overflow-y-hidden">
+		<!-- Trick to avoid the scroll bar to appear on the right when switching to full screen -->
+		<Collapse :when="!is_full_screen">
+			<Toolbar class="w-full border-0 h-14">
+				<template #start>
+					<Button icon="pi pi-angle-left" class="mr-2 border-none" severity="secondary" text @click="goBack" />
+				</template>
+
+				<template #center>
+					{{ $t("favourites") }}
+				</template>
+
+				<template #end> </template>
+			</Toolbar>
+		</Collapse>
+		<div
+			id="galleryView"
+			class="relative flex flex-wrap content-start w-full justify-start overflow-y-auto"
+			:class="is_full_screen ? 'h-svh' : 'h-[calc(100vh-3.5rem)]'"
+			v-on:scroll="onScroll"
+		>
+			<PhotoThumbPanel
+				v-if="layoutConfig !== undefined && photos.length > 0"
+				header="gallery.album.header_photos"
+				:photos="photos"
+				:album="undefined"
+				:gallery-config="layoutConfig"
+				:photo-layout="'square'"
+				:selected-photos="selectedPhotosIds"
+				:is-timeline="false"
+				:with-control="false"
+				@clicked="photoClick"
+			/>
+		</div>
+	</div>
+	<!-- @clicked="photoClick"
+    @contexted="photoMenuOpen" -->
+</template>
+<script setup lang="ts">
+import PhotoThumbPanel from "@/components/gallery/albumModule/PhotoThumbPanel.vue";
+import { useScrollable } from "@/composables/album/scrollable";
+import { usePhotoRoute } from "@/composables/photo/photoRoute";
+import { useSelection } from "@/composables/selections/selections";
+import { useGetLayoutConfig } from "@/layouts/PhotoLayout";
+import { useFavouriteStore } from "@/stores/FavouriteState";
+import { useTogglablesStateStore } from "@/stores/ModalsState";
+import { shouldIgnoreKeystroke } from "@/utils/keybindings-utils";
+import { onKeyStroke } from "@vueuse/core";
+import { storeToRefs } from "pinia";
+import Button from "primevue/button";
+import Toolbar from "primevue/toolbar";
+import { ref } from "vue";
+import { computed } from "vue";
```

Comment:
computed, onMounted can set in the same import

Reply:
[7ad3073](https://github.com/LycheeOrg/Lychee/pull/3049/commits/7ad307389239a77f87044ccd32a8efff70e2dfb6)